ktor-sample/src/main/kotlin/novatec/info/plugins/Routing.kt

```diff
@@ -0,0 +1,16 @@
+package novatec.info.plugins
+
+import io.ktor.server.routing.*
+import io.ktor.server.response.*
+import io.ktor.server.application.*
+import novatec.info.model.AuthorRepository
+
+fun Application.configureRouting(authorRepository: AuthorRepository) {
+    routing {
+        get("/authors") {
+            val authors = authorRepository.getAllAuthors()
+            println(authors)
```

Comment:
I think this `println()` slipped without being noticed 😉

Reply:
exactly
thanks for finding